src/components/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useState, ChangeEvent, DragEvent, useRef } from 'react'
+import styled from '@emotion/styled'
+import { css } from '@emotion/react'
+import {
+  color,
+  space,
+  radius,
+  transition,
+  fontWeight,
+  device,
+} from '../../theme'
+import { H4 } from '../Heading'
+import { SmallText } from '../Text'
+import { BaseButton } from '../BaseButton'
+import ILLUSTRATION from './files.svg'
```

Comment:
I dont think the svg is centered 🤔 
![image](https://user-images.githubusercontent.com/49148610/188564281-05d389b0-f331-41fe-b12d-553616427bb4.png)

Reply:
Nice catch! Fixed 🙏